### ISSUE TITLE
Can't install io.js on x86 (ia32)

### ISSUE THREAD

**Issue by hoshi-takanori** [hokaccha/nodebrew#34]
```
$ uname -a
Linux fit-pc2 3.2.0-4-686-pae #1 SMP Debian 3.2.65-1+deb7u1 i686 GNU/Linux
$ nodebrew help | head -1
nodebrew 0.8.0
$ nodebrew install-binary io@v1.0.2
fetch: https://iojs.org/download/release/v1.0.2/iojs-v1.0.2-linux-x86.tar.gz
######################################################################## 100.0%
Error: No such file or directory at /home/hoshi/.nodebrew/current/bin/nodebrew line 141.
$ ls -l .nodebrew/src/v1.0.2
total 7428K
drwxr-xr-x 6 hoshi hoshi    4096 Jan 17 09:02 iojs-v1.0.2-linux-ia32/
-rw-r--r-- 1 hoshi hoshi 7586403 Jan 17 09:02 iojs-v1.0.2-linux-x86.tar.gz
$ 
```

`mv .nodebrew/src/v1.0.2/iojs-v1.0.2-linux-ia32 .nodebrew/iojs/v1.0.2` seems to work.

**Comment by hokaccha** [comment]
Thank you for reporting. I could reproduce the issue, so I will work it.

**Comment by hokaccha** [comment]
This issue is fixed at v0.8.1. Try it!

```
$ nodebrew selfupdate
$ nodebrew install-binary io@v1.0.2
```
